packages/terra-application-menu-layout/tests/jest/ApplicationMenu.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import ApplicationMenuLayout from '../../src/ApplicationMenuLayout';
+
+describe('ApplicationMenuLayout', () => {
+  it('should render without optional props', () => {
+    const result = shallow(<ApplicationMenuLayout />);
+    expect(result).toMatchSnapshot();
+  });
+
+  it('should render with provided props', () => {
+    const result = shallow(
+      <ApplicationMenuLayout
```

Comment:
Same question as above regarding adding additional tests for different props provided

Reply:
Updated.